### PR TITLE
DM-33150: Remove Gen 2 support from ap_verify

### PR DESCRIPTION
**Pull request by kfindeisen**
This PR replaces the Gen 2 stub content in `ap_verify_dataset_template` with Gen 3 stubs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,27 +13,24 @@ path                  | description
...
+`preloaded`           | To be populated with a Gen 3 Butler repository (see below). This repository must never be written to; instead, it should be copied to a separate location (this is handled automatically by `ap_verify`, see below).
 `dataIds.list`        | List of dataIds in this repo. For use in running Tasks. Currently set to run all Ids.
 
 
 Gen 3 Collections
 -----------------
 
-The Gen 3 repository in `preloaded/` must contain the following collections.
-These may duplicate the directories above to support both Gen 2 and Gen 3 processing.
+The Gen 3 repository in `preloaded/` must contain the following collections; these may be chained collections wrapping nonstandard names.
```
Comment (parejkoj) [inline README.md]: "wrapping nonstandard names"? Do you mean "these may be chained collections containing collections with other names" (or something like that)?

File: pipelines/ApVerifyWithFakes.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: Stub instrumented Alert Production pipeline
+#
+# This pipeline does not depend on the local ApVerify.yaml, because the definition
+# of the primary ApVerifyWithFakes.yaml is more likely to change than the
+# data-specific overrides, and importing both pipelines can't merge changes to
+# the same task.
```
Comment (parejkoj) [inline pipelines/ApVerifyWithFakes.yaml:6]: "...importing both pipelines can't merge changes to the same task." - I'm having trouble parsing this.

File: README.md [hunk 3]
```diff
@@ -13,27 +13,24 @@ path                  | description
...
-`preloaded`           | To be populated with a Gen 3 Butler repository (see below). This repository must never be written to; instead, it should be copied to a separate location (this is handled automatically by `ap_verify`, see below). Currently empty, but can be populated from the other directories using `scripts/add_gen3_repo.py`.
-`refcats`             | To be populated with tarball(s) of HTM shards from relevant reference catalogs. Currently contains a small (useless) example tarball.
+`config`              | To be populated with dataset-specific configs. Currently empty.
+`pipelines`           | To be populated with dataset-specific pipelines. Currently contains three example files specialized for ImSim data.
+`preloaded`           | To be populated with a Gen 3 Butler repository (see below). This repository must never be written to; instead, it should be copied to a separate location (this is handled automatically by `ap_verify`, see below).
```
Comment (parejkoj) [inline README.md]: "to a separate location before use"?

File: README.md (hunk 1 above)
Comment (kfindeisen) [reply README.md]: Something like that. Basically, a rule saying that runs must follow any given naming scheme would make it almost impossible to create or update such a repository in the first place.

File: pipelines/ApVerifyWithFakes.yaml (hunk 2 above)
Comment (kfindeisen) [reply pipelines/ApVerifyWithFakes.yaml:6]: If you try to import two pipelines (like `dataset/pipelines/ApVerify.yaml` and `ap_verify/pipelines/ApVerifyWithFakes.yaml`) and they both override the same task, Middleware won't let you do it (IIRC). You definitely won't get the same behavior as with multiple non-pipeline config overrides.

File: README.md (hunk 1 above)
Comment (parejkoj) [reply README.md]: I'm not sure if there's a way to reword it?

File: pipelines/ApVerifyWithFakes.yaml (hunk 2 above)
Comment (parejkoj) [reply pipelines/ApVerifyWithFakes.yaml:6]: I wish I could come up with a rewording that would make more sense to me...

File: README.md (hunk 1 above)
Comment (kfindeisen) [reply README.md]: "chained collections containing arbitrarily-named runs"?

Comment (parejkoj) [reply README.md]: Oh, yeah, I like that! Thank you.

**Pull request merged**